src/components/AnnotationEditor.jsx

```diff
@@ -63,15 +77,31 @@ class AnnotationEditor extends React.Component {
         </Tabs>
         <RaisedButton
           label="Cancel"
-          secondary
           onTouchTap={this.clearAnnotation}
+          secondary
         />
         <RaisedButton
-          label="Save"
-          primary
           disabled={!this.state.annotation}
+          label="Save"
           onTouchTap={this.saveAnnotation}
+          primary
         />
+        <a
+          href="http://commonmark.org/help"
+          style={styles['markdown-indicator']}
+          target="_blank"
+        >
+          <img
+            alt="Markdown Logo"
+            src={markdownLogo}
+            style={styles['markdown-logo']}
```

Comment:
I think `styles.markdown-logo` is better, this seems to indicate a key-value model.